### ISSUE TITLE
FEATURE: ability to view how many augments are remaining for factions with pending invite

### ISSUE THREAD

**Issue by maconard** [bitburner-official/bitburner-src#325]
## Description

Currently the pending faction invites show you what type of jobs/work they have available and any faction v faction limitations, but it doesn't show you how many augments you have left with them -- it only shows you that after you join. It would, however, be incredibly useful information before joining, and I don't (personally) see a good reason to hide this information, seeing as it just forces the user to cross reference their augment list against a wiki or something.

## Version
Bitburner v2.2.1 (c46cedd5), Steam

## Screenshot
Pending invites:
![image](https://user-images.githubusercontent.com/31703304/212597463-1f54976b-ec2e-471c-b013-2fdf56c97acd.png)

Joined factions:
![image](https://user-images.githubusercontent.com/31703304/212597496-5b98c82d-9741-4277-be1c-bcf2ce052790.png)

Cheers!

**Comment by Zelow79** [comment]
This already exists in singularity easily enough though. I think it defeats the purpose to have this function personally.
![image](https://user-images.githubusercontent.com/32428876/213847953-3746e682-c75d-4333-93e8-cffb8b437151.png)
![image](https://user-images.githubusercontent.com/32428876/213847966-1242cee5-a7ff-4cb1-93aa-09ee296a647b.png)
TRP is the only augment I don't have.

Update: Added a bit more to the code to hopefully make it more understandable. I will also share the code so it can be copied and pasted, if you wanna test it yourself below. *you can uncomment the owned augs line if you wanna see it, I have every augment so it was just spam for me to use it.
![image](https://user-images.githubusercontent.com/32428876/213848940-e9a89bc7-5d0d-4e16-a514-0287f7a70fd3.png)
This would be called with `run deleteMe.js Daedalus`
```
export async function main(ns) {
	ns.clearLog(); ns.tail();
	const x = ns.singularity.getAugmentationsFromFaction(ns.args[0]);
	const y = ns.singularity.getOwnedAugmentations(true);
	const filter = x.filter(x => !y.includes(x));
	//ns.print(`All currently owned Augs: ${y}`);
	ns.print(`All Augs ${ns.args[0]} offers: ${x}`);
	ns.print(`What you have left to get: ${filter}`);
}

**Comment by Snarling** [comment]
Fixed by #357
